src/lib/browse/data.ts

```diff
@@ -35,17 +34,12 @@ export function processInput(
   }
 
   for (let feature of gj.features) {
-    schemes.get(feature.properties!.scheme_reference).num_features++;
```

Comment:
I cleaned up these problems in the latest version of the scheme file I've sent around